--- origsrc/qimageblitz-0.0.4/blitz/asm_scale.S	2007-11-07 05:58:44.000000000 -0600
+++ src/qimageblitz-0.0.4/blitz/asm_scale.S	2008-07-23 01:28:07.890625000 -0500
@@ -8,7 +8,7 @@
 	.___end_##sym:;                        \
 	.size ___##sym,.___end_##sym-___##sym; \
 	.align 8;
-#elif defined(__APPLE__)
+#elif defined(__APPLE__) || defined(__CYGWIN__)
 #define PR_(foo) ___##foo
 #define PT_(foo,func) ___##foo,##func
 #define SIZE(sym)                           \
